Update the README with documentation on how to use this (#30)
Lullabot/drainpipe: README.md

# Drainpipe

Drainpipe is a composer package which provides a number of build tool helpers
for a Drupal site, including:

- Site and database updates

> - Artefact packaging for deployment to a hosting provider

Reviewer: ```suggestion
- Artifact packaging for deployment to a hosting provider
```

> - Test setups

Reviewer: ```suggestion
- Automated testing setup with support for PHPUnit and Nightwatch tests
```

## Installation

```
composer require lullabot/drainpipe

> composer require lullabot/drainpipe-dev --dev

Reviewer: ```suggestion
# Includes development dependencies, but only in the `require-dev` section. This step is required for Drainpipe to function correctly.
composer require lullabot/drainpipe-dev --dev
```

Unless it's actually optional, in which case we should call it out!

Author: It's not required for it to functional correctly!

```

> There are a number of pre-configurations provided if you're using
> [DDEV](https://ddev.readthedocs.io/en/stable/), so either set DDEV up first
> before requiring this project, or run `composer update` if DDEV is added later.

Reviewer: ```suggestion
Drainpipe integrates with [DDEV](https://ddev.readthedocs.io/en/stable/), but
will only add the relevant files when DDEV is detected in the repository. Either
set DDEV up first before requiring this project, or run `composer update` if
DDEV is added later.
```

```
composer create-project drupal/recommended-project drupal
cd drupal
ddev config
composer require lullabot/drainpipe
composer require lullabot/drainpipe-dev --dev
ddev start
```

## Usage

> Helpers are provided as [Taskfiles](https://taskfile.dev/#/). A full list of

Reviewer: ```suggestion
Build tasks are provided as [Taskfiles](https://taskfile.dev/#/). A full list of
```

# Drainpipe

Drainpipe is a composer package which provides a number of build tool helpers
for a Drupal site, including:

- Site and database updates
- Artifact packaging for deployment to a hosting provider
- Automated testing setup with support for PHPUnit and Nightwatch tests

## Installation

```
composer require lullabot/drainpipe
# Includes development dependencies, but only in the `require-dev` section. This step is required for Drainpipe to provide test helpers.
composer require lullabot/drainpipe-dev --dev
```

Drainpipe integrates with [DDEV](https://ddev.readthedocs.io/en/stable/), but
will only add the relevant files when DDEV is detected in the repository. Either
set DDEV up first before requiring this project, or run `composer update` if
DDEV is added later.

```
composer create-project drupal/recommended-project drupal
cd drupal

> ddev config
> composer require lullabot/drainpipe
> composer require lullabot/drainpipe-dev --dev
> ddev start

Author: ```suggestion
ddev config
ddev start
ddev composer require lullabot/drainpipe
ddev composer require lullabot/drainpipe-dev --dev
# Restart is required to enable the provided Selenium containers
ddev restart
```
